packages/suite/src/actions/suite/initAction.ts

```diff
@@ -74,7 +76,31 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     try {
         // it is necessary to unwrap the result here because init calls async thunk from redux-toolkit which is always resolved
         // see more details here: https://redux-toolkit.js.org/api/createAsyncThunk#unwrapping-result-actions
-        await dispatch(trezorConnectActions.connectInitThunk()).unwrap();
+        await dispatch(
+            trezorConnectActions.connectInitThunk({
+                [DEVICE.CONNECT]: (_device, prevConnectedDevices) => {
+                    const previouslyConnectedDevices = prevConnectedDevices.filter(
+                        device => device.connected,
+                    );
+                    const currentConnectedDevices = selectDevices(getState()).filter(
+                        device => device.connected,
+                    );
+
+                    if (
```

Comment:
It would be cool if I could say that I don't want have the `device connected` triggered "on initial" run. when the connect is initialized for the first time.

Unfortunatel,y there isn't yet such thing.

Reply:
Such thing would an internal state of the connect, it would be cool to have a method for it.